contracts/rbn-staking/PenaltyDistributor.vy

```diff
@@ -415,14 +430,35 @@ def donate(_amount: uint256) -> bool:
     @return bool success
     """
     assert _amount != 0
+    assert msg.sender == self.voting_escrow #dev: must be voting escrow
```

Comment:
do we need this assert?

We are using tx.origin which is generally unsafe, but unsure if there can be issue since funds are sent to tx.origin

Reply:
Why did we remove this in the end? Seems like a reasonable guard to me to prevent phishing attacks